policyengine_us/variables/household/demographic/tax_unit/tax_unit_is_dependent.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_is_dependent(Variable):
```

Comment:
Is this this? https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/household/demographic/tax_unit/dsi.py